aibolit/metrics/RFC/rfc.py

```diff
@@ -0,0 +1,51 @@
+from aibolit.utils.ast import AST, ASTNodeType
+from aibolit.utils.java_package import JavaPackage
+
+
+class RFC:
+    '''
+    The Response For a Class is an object-oriented
+    metric that shows the interaction of the class'
+    methods with other methods.
+    '''
+    def __init__(self):
+        pass
+
+    def value(self, filename: str) -> int:
+        class_methods = set()
+        called_methods = set()
+        initialized_method_vars = set()
+
+        p = JavaPackage(filename)
+        for class_name in p.java_classes:
+            tree = p.java_classes[class_name]
+            for class_method in tree.subtrees_with_root_type(ASTNodeType.METHOD_DECLARATION):
+                ast_each_method = AST(tree.tree.subgraph(class_method), class_method[0])
+                # to form a set of all methods in the class
+                names = list(ast_each_method.children_with_type(ast_each_method.root, ASTNodeType.STRING))
+                method_name = tree.get_attr(names[0], 'string')
```

Comment:
> tree.get_attr(names[0], 'string')
@aravij  It's not usual that to get the name of the method we have to call the method of the some AST tree

Reply:
rewrrited